src/postgresql.tf

```diff
@@ -9,13 +9,13 @@ module "postgresql_snet" {
 }
 
 data "azurerm_key_vault_secret" "db_administrator_login" {
-  count        = var.prostgresql_enabled ? 1 : 0
+  count        = var.postgresql_enabled ? 1 : 0
```

Comment:
Do we really need this count .... why not just create the resource in every environment?

Reply:
unfortunately in this case yes.
In the dev environment we'll have a single server instance (used by other applications such as the mock psp too cc @infantesimone ) while in uat / prod we'll go towards a [Flexible solution](https://pagopa.atlassian.net/wiki/spaces/PAG/pages/474219118/Migrazione+DB+Oracle+su+Cloud+Azure#Tipologia-di-DB-PaaS) cc @pasqualedevita 

I changed the condition removed `postgresql_enabled`  feature flag and user `env_short` clearly
do u agree @uolter  ?